Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocTextValue.css

```diff
@@ -45,48 +45,28 @@ span.bocTextValue .content
   display: inline-block;
 }
 
-.bocTextValue.readOnly .content span[tabindex]:first-child
-{
-  /* Ensures that diacritic marks can be rendered without being cut off. */
-  padding-top: 0.2em;
-}
-
-.bocTextValue.readOnly .content span[tabindex]:last-child
-{
-  /* Ensures that diacritic marks can be rendered without being cut off. */
-  padding-bottom: 0.2em;
-}
-
 .bocTextValue.readOnly .content span[tabindex]:empty
 {
-  /* Ensures that empty readonly values are not collapsed and can therefor visualize the keyboard focus */
-  width: calc(100% - 2px);
+  /* Ensures that empty readonly values are not collapsed and can therefore visualize the keyboard focus */
+  width: 100%;
   min-height: 1.5em;
-  padding-top: 0;
-  padding-bottom: 0;
-  vertical-align: -0.2em;
+  vertical-align: middle;
 }
 
 span.bocTextValue input
 {
   width: 100%;
+  min-height: inherit;
```

Comment:
why inherit?

Reply:
Not necessary, 100% works too. One of the two is needed, though, to fill the parent.

Reply:
Then `100%` it is. `inhert` doesn't mean 'copy the px count but instead, it's a reapply-the-parent's-style-here-because-my-current-style-was-overridden-and-I-want-the-parent-back.